res/controllers/Pioneer-DDJ-FLX4.midi.xml

```diff
@@ -0,0 +1,3228 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxMIDIPreset schemaVersion="1" mixxxVersion="2.3">
+    <info>
+        <name>Pioneer DDJ-FLX4</name>
+        <author>Warker/nschloe/dj3730/jusko/Robert904</author>
```

Comment:
```suggestion
        <author>Warker/nschloe/dj3730/jusko/Robert904</author>
```
Just mention the author of this mapping - it's enough to mention, that it's based on the DDJ-400